### PR TITLE
ToH 1: adjust introduction of angular_forms

### LINKED ISSUE #1048: Fix tutorial's discussion of angular_forms & the directives list
See #1041 for details.

### PR DESCRIPTION
**Pull request by chalin**
The updated prose highlights the fact that the template parser will report and error, and then explains how to fix it (with steps in a reasonable order).

Fixes #1048

Staged at https://angulardart-org-dev3.firebaseapp.com/angular/tutorial/toh-pt1

### CODE REVIEW

File: src/angular/tutorial/toh-pt1.md [hunk 1]
```diff
@@ -148,20 +148,48 @@ and from the textbox back to the property.
   [Template Syntax](../guide/template-syntax.html#ngModel) pages.
 </div>
 
-### @Component(directives: ...)
+## Declare non-core directives
```
Comment (kwalrath) [inline src/angular/tutorial/toh-pt1.md]: I like this heading. Let's change the other headings to match, as much as possible:

Create a Hero class
Add HTML...
Use two-way binding

"Edit the hero name" isn't an instruction to the reader, so maybe change it to:

Enable editing the hero name

File: src/angular/tutorial/toh-pt1.md [hunk 2]
```diff
@@ -148,20 +148,48 @@ and from the textbox back to the property.
...
+### Oops, unknown directive!
+
+<i class="material-icons">open_in_browser</i>
+If you **refresh the browser,** the app won't load.
+To know why, look at the `pub serve` output. The template
 compiler doesn't recognize `ngModel`, and issues a parse error for
 `AppComponent`:
 
-```
+```nocode
```
[the remark this answers is not in the document]
Comment (chalin) [reply src/angular/tutorial/toh-pt1.md]: Yes: https://github.com/google/code-prettify#how-do-i-prevent-a-portion-of-markup-from-being-marked-as-code

File: src/angular/tutorial/toh-pt1.md (hunk 1 above)
Comment (chalin) [reply src/angular/tutorial/toh-pt1.md]: Done. And offline we agreed to shorten the "Add HTML..." to "Add multi-line template HTML".

File: src/angular/tutorial/toh-pt1.md [hunk 3]
```diff
@@ -148,20 +148,48 @@ and from the textbox back to the property.
...
 </div>
 
-### @Component(directives: ...)
+## Declare non-core directives
 
 Unfortunately, immediately after this change, the **app breaks**!
-If you open the browser console and refresh the page, you'll see Angular
-complaining.  To know why, look at the `pub serve` output. The template
+
+### Oops, unknown directive!
```
[the remark this answers is not in the document]
Comment (chalin) [reply src/angular/tutorial/toh-pt1.md]: done

**Pull request merged**